### DISCUSSION TITLE
How to switch suggestions in copilot.vim

### DISCUSSION THREAD

**Discussion by Lucklyric** [community/community#7082] [Copilot Conversations] [answered]
In general, copilot.vim performs admirably. I'm wondering how I can switch between different suggestions like in VSCode.

**Comment by Lucklyric** [comment, marked as answer]
seems duplicated with another one. This can be ignored.

**Reply by leahein** [reply]
Duplicate [here](https://github.com/github/feedback/discussions/7053)
